src/common.js

```diff
@@ -259,10 +259,9 @@ function wrap(cmd, fn, options) {
     try {
       var args = [].slice.call(arguments, 0);
 
+      // Log the command to stderr, if appropriate
       if (config.verbose) {
-        args.unshift(cmd);
-        console.error.apply(console, args);
-        args.shift();
+        console.error.apply(console, [cmd].concat(args));
```

Comment:
I realized this could be a one-liner, so I went ahead and made the change. This doesn't affect the behavior.